uwsgi/public.wsgi

```diff
@@ -54,6 +54,10 @@ if os.environ.get("AUTOGRAPH_URL"):
 
 
 cache.make_cache("blob", 500, 3600)
+cache.make_cache("releases", 500, 3600)
+cache.make_cache("releases_data_version", 500, 60)
+cache.make_cache("release_assets", 500, 3600)
+cache.make_cache("release_assets_data_versions", 5000, 60)
```

Comment:
I thought about adding caches on admin (we obviously do have them for the old tables), but I couldn't find a compelling reason we do so for the old table, nor think of a reason why they'd be especially helpful here. Most accesses of a Release on admin are to update them when we're building a Release, which means we just invalidate over and over and over. Even casual usage of the UI wouldn't make use of them, since you have to click into `View` or `Update` a Release to see the blob. I'm open to other interpretations of this though.

Reply:
I can't think of any reasons either, thanks for the explanation.